langlib/lang.value/src/main/ballerina/value.bal

```diff
@@ -153,6 +196,12 @@ public isolated function toBalString(any v) returns string = @java:Method {
 # The subset of Ballerina expression syntax supported is that produced
 # by toBalString when applied to an anydata value.
 #
+# ```
+# "12.12d".fromBalString() ⇒ 12.12
+#
+# "[1, 2, error(\"Error\"))]".fromBalString() ⇒ error
```

Comment:
We can use something like `"[1, 2, !]".fromBalString()` to be less complicate,
wdyt?